main.go

```
@@ -732,15 +723,17 @@ to get your bearings.
 		}
 		v := connect()
 		for _, path := range args {
-			if opt.Delete.Recurse {
+			_, key := vault.ParsePath(path)
+			//Ignore -R if path has a key because that makes no sense
+			if opt.Delete.Recurse && key == "" {
 				if !opt.Delete.Force && !recursively("delete", args...) {
 					return nil /* skip this command, process the next */
 				}
 				if err := v.DeleteTree(path); err != nil {
 					return err
 				}
 			} else {
-				if err := v.Delete(path); err != nil {
+				if err := v.Delete(path); err != nil && !vault.IsNotFound(err) {
```

Why do we now ignore errors when trying to delete things that aren't found?